app/routes/workload-points.js

```diff
@@ -33,7 +34,39 @@ module.exports = function (router) {
           breadcrumbs: result.breadcrumbs,
           wp: result.workloadPoints,
           updatedBy: result.updatedBy,
-          successText: successText
+          successText: successText,
+          isT2A: false
```

Comment:
Should this always be false?

Reply:
This route is specifically for non-T2A workload points and so it should always be false.